woocommerce-paypal-payments.php

```diff
@@ -140,3 +141,19 @@ function( $links ) {
 	);
 
 } )();
+
+
+/**
+ * Add woocommerce_inbox_variant for the Remote Inbox Notification.
+ *
+ * P2 post can be found at https://wp.me/paJDYF-1uJ.
+ */
+if ( ! function_exists( 'add_woocommerce_inbox_variant' ) ) {
+	function add_woocommerce_inbox_variant() {
+		$config_name = 'woocommerce_inbox_variant';
```

Comment:
This should be `$option` to match `get_option` and `update_option`'s signatures

Reply:
Updated in https://github.com/woocommerce/woocommerce-paypal-payments/pull/160/commits/aac19bb8a1173e2e5fa736ddbad5e9077b296dc7